src/test/rpc/AccountObjects_test.cpp

```diff
@@ -1032,13 +1032,139 @@ class AccountObjects_test : public beast::unit_test::suite
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::hashes), 0));
     }
 
+    void
+    testNFTsMarker()
+    {
+        testcase("NFTsMarker");
```

Comment:
it might be useful to have some comments in the beginning to describe the what this test is trying to achieve from a high level